plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Applications.kt

```diff
@@ -0,0 +1,16 @@
+package com.github.triplet.gradle.play.internal
+
+import java.io.File
+import java.io.FileFilter
+
+internal object ApkFileFilter : FileFilter {
+    private const val APK_EXTENSION = "apk"
+
+    override fun accept(file: File) = file.extension.toLowerCase() == APK_EXTENSION
+}
+
+internal object BundleFileFilter : FileFilter {
```

Comment:
What do we do with this `BundleFileFilter`? Or is this included for an upcoming PR?